doc/deployment_guide.md

```diff
@@ -0,0 +1,1228 @@
+﻿
+<img src="media/dc99730255437cdd115b74c45cc6c19d.jpg" alt="drawing" width="200px"/>
+
+# Dell EMC Red Hat Ready Architecture Deployment Guide Notes
+### Version 13.0
+##### Dell EMC Service Provider Solutions
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<br><br><br><br><br><br><br><br><br>
+<div style="page-break-after: always;"></div>
+
+# Trademarks
+Copyright © 2014-2018 Dell Inc. or its subsidiaries. All rights reserved.
+
+Microsoft® and Windows® are registered trademarks of Microsoft Corporation in the United States and/or other countries.
+
+Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered trademarks of Red Hat, Inc., registered in the U.S. and other countries. Linux® is the registered trademark of Linus Torvalds in the U.S. and other countries. Oracle® and Java® are registered trademarks of Oracle Corporation and/or its affiliates.
+
+DISCLAIMER: The OpenStack® Word Mark and OpenStack Logo are either registered trademarks/ service marks or trademarks/service marks of the OpenStack Foundation, in the United States and other countries, and are used with the OpenStack Foundation's permission. We are not affiliated with, endorsed or sponsored by the OpenStack Foundation or the OpenStack community.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<div style="page-break-after: always;"></div>
+
+# Contents:
+
+[Chapter 1 : Overview](#Chapter-1-:-Overview)
+
+[Chapter 2 : Red Hat Subscriptions](#Chapter-2-:-Red-Hat-Subscriptions)
+
+[Chapter 3 : Automation Configuration Files](#Chapter-3-:-Automation-Configuration-Files)
+
+[Chapter 4   Preparing and Deploying the Solution Admin Host](#Chapter-4-Preparing-and-Deploying-the-Solution-Admin-Host)
+
+[Chapter 5 : Deploying the Undercloud and the OpenStack Cluster](#Chapter-5-:-Deploying-the-Undercloud-and-the-OpenStack-Cluster)
+
+[Appendix A : Files References](#Appendix-A-:-Files-References)
+
+[Appendix B : Updating RPMs on Version Locked Nodes](#Appendix-B-:-Updating-RPMs-on-Version-Locked-Nodes)
+
+[Appendix C : OpenStack Operations Functional Test](#Appendix-C-:-OpenStack-Operations-Functional-Test)
+
+

```

Comment:
Only chapter 4 is a link sending to the target content, the rest of them just text ?

Reply:
final document is now ready

Reply:
fixed